site3/website/docs/reference/config.md

```diff
@@ -226,6 +226,17 @@ The table below lists parameters that you can set to configure bookies. All conf
 | dbStorage_rocksDB_numFilesInLevel0 |  | 10 | 
 | dbStorage_rocksDB_maxSizeInLevel1MB |  | 256 | 
 
+> Note: 
```

Comment:
@nicoloboschi IIUC, this only affects the NEXT version of the website, right?

Reply:
yes